beacon_node/src/config.rs

```diff
@@ -116,7 +116,14 @@ pub fn get_config<E: EthSpec>(
     }
 
     if cli_args.is_present("http-disable-legacy-spec") {
-        client_config.http_api.serve_legacy_spec = false;
+        warn!(
+            log,
+            "The flag --http-disable-legacy-spec is deprecated and will be removed"
```

Comment:
Perhaps you want to point users towards `--http-spec-fork`? I'm easy, just a suggestion :)

Reply:
they're pretty much unrelated, I reckon we leave it